widelands.yaml

```diff
@@ -0,0 +1,54 @@
+name: widelands
+buildsystem: cmake-ninja
+builddir: true
+config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  - -DCMAKE_INSTALL_PREFIX=/app/bin
+  - -DWL_INSTALL_BASEDIR=/app/share/widelands
+  - -DWL_INSTALL_DATADIR=/app/share/widelands
+  - -DBOOST_ROOT=/app
+  - -DGLEW_ROOT=/app
+  - -DGLEW_GLEW_LIBRARY=/app/lib/libGLEW.so
+  - -DGLEW_INCLUDE_DIR=/app/include/GL
+  # https://wl.widelands.org/forum/topic/1343/?page=3 & https://wl.widelands.org/forum/post/9857/
+  - -DGLEW_LIBRARY:FILEPATH=/app/lib/libGLEW.a
+  - -DOPENGL_glu_LIBRARY=/app/lib/libGLU.a
+  - -DOPTION_BUILD_WEBSITE_TOOLS=OFF
+sources:
+  # Source0
+  - type: archive
+    url: https://launchpad.net/widelands/build19/build19/+download/widelands-build19-src-gcc7.tar.bz2
+    sha256: ef73b8cf5a7c680bad5caade907f1cc6ee88dd3729470b9214a3c72658585458
+  # Source1 - Numix-Circle - numix-icon-theme-circle-0.1.0-13.git5a11140.el7.src.rpm
```

Comment:
???

Reply:
> ???

I borrowed this SVG icon from the Numix-Circle theme.